ssl/record/rec_layer_s3.c

```diff
@@ -743,6 +743,16 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
         s->s3->empty_fragment_done = 1;
     }
 
+    if (BIO_get_offload_tx(s->wbio)) {
+        /*
+         * ktls doesn't modify the buffer, but to avoid a warning we need to
+         * discard the const qualifier
+         */
+        SSL3_BUFFER_set_buf(&s->rlayer.wbuf[0], (unsigned char *)buf);
```

Comment:
Won't this leak any existing buffer?

Reply:
Right. I haven't thought of that. I'll add a ssl3_release_write_buffers call to release the existing buffers before we start using ktls to ensure all previous buffers are released.

Reply:
Done.